test/functional/sqlsrv/MsHelper.inc

```diff
@@ -330,7 +330,7 @@ function getSeqPlaceholders($num)
  */
 function isColEncrypted()
 {
-    global $keystore, $dataEncrypted;
+    global $keystore;
```

Comment:
since you are changing this here can you fix - I really hate stupid unnecessary code like this. This is just
return $keystore != KEYSTORE_NONE
!!